docs/sources/write/image-guidelines/index.md

```diff
@@ -142,6 +142,24 @@ Consult the following guidelines when you create screenshots:
 - **Personal identifiable information (PII)**: Make sure to mask, modify, or remove any PII such as passwords, logins, account details, or other information that could compromise security.
 - **Alt text**: Make sure to include [alt text](#alt-text) for every image.
 
+Refer to the next section for examples of good screenshots.
+
+#### Screenshot examples
+
+Good example of size at 545 pixels wide:
+![Good size example](/media/docs/grafana/panels-visualizations/screenshot-stat-single-value-v11.3.png)
```

Comment:
Note this image looks a little blurry on high dpi screens.

In this case I would suggest a file that is 2x resolution, or 1090px wide, and then limited to 545px wide with the figure shortcode's `max-width` argument.

Reply:
I updated the image and booked a meeting with you to go over my confusion around this stuff!